cmd/elasticindexer/main.go

```diff
@@ -104,6 +108,19 @@ func startIndexer(ctx *cli.Context) error {
 	return nil
 }
 
+func requestSettings(host wsindexer.WSClient, retryDurationInSec uint32) {
+	for {
+		err := host.Send([]byte{}, outport.TopicSettings)
+		if err != nil {
+			time.Sleep(time.Duration(retryDurationInSec) * time.Second)
+
+			log.Debug("request settings", "error", err)
```

Comment:
might print before the `time.Sleep` and also display a `will retry`: 
- `log.Debug("request settings", "error", err)`
=>
- `log.Debug("unable to request settings - will retry", "error", err)`

Reply:
done.